source/stylesheets/utilities/_spacing.scss

```diff
@@ -36,6 +36,14 @@ $spacings: (
     }
   }
 
+  .u-#{$name}PushRightAlways {
+    margin-right: $value;
+  }
+
+  .u-#{$name}PushLeftAlways {
+    margin-right: $value;
+  }
```

Comment:
PushRight is only used with the `small` variant, and PushLeft is not even used at all.

I would argue it's better to just declare `.u-smallPushRightAlways` outside the loop, instead of adding so many useless rules to the stylesheet